src/bulkOperationCommand.ts

```diff
@@ -60,6 +61,10 @@ export abstract class BulkOperationCommand extends BulkBaseCommand {
       default: false,
       exclusive: ['wait'],
     }),
+    verbose: Flags.boolean({
+      summary: messages.getMessage('flags.verbose'),
+      default: false,
```

Comment:
the default value for Boolean flags is `false` can you remove this line?

Reply:
Sure, will remove this in the next commit 👍